clients/cloud/csharp/Program.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Confluent.Kafka;
+using Confluent.Kafka.Admin;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace CCloud
+{
+    class Program
+    {
+        static string ExtractJaasValue(string jaasConfig, string key)
+        {
+            var beginToken = key + "\\=\"";
+            var startIdx = jaasConfig.IndexOf(beginToken);
+            var endIdx = jaasConfig.IndexOf("\"", startIdx + beginToken.Length);
+            return jaasConfig.Substring(startIdx + beginToken.Length, endIdx - startIdx - beginToken.Length);
+        }
+
+        static async Task<ClientConfig> LoadConfig(string configPath, string certDir)
+        {
+            try
+            {
+                var cloudConfig = (await File.ReadAllLinesAsync(configPath))
+                    .Where(line => !line.StartsWith("#"))
+                    .ToDictionary(
+                        line => line.Substring(0, line.IndexOf('=')),
+                        line => line.Substring(line.IndexOf('=') + 1));
+
+                var clientConfig = new ClientConfig
+                {
+                    BootstrapServers = cloudConfig["bootstrap.servers"].Replace("\\", ""),
+                    BrokerVersionFallback = "0.10.0.0",

```

Comment:
This and FallbackMs are not required with librdkafka v1.0.0

Reply:
awesome